sdk/devcenter/azure-developer-devcenter/samples/create_devbox_sample.py

```diff
@@ -47,12 +40,13 @@
     1) DEVCENTER_ENDPOINT - the endpoint for your devcenter
 """
 
-def get_project_name(LOG, client):
-    projects = list(client.projects.list_by_dev_center(top=1))
-    return projects[0].name
 
+def create_dev_center_client():
```

Comment:
You can move this to its own file since you also create a client in the function below and creating the dev box is the more interesting part of what we're doing here.